utils/related_news.py

```
@@ -0,0 +1,46 @@
+import numpy as np
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)